src/AgentBuilder.php

```diff
@@ -37,20 +37,20 @@ public function withEventCollectors(Collection $collectors): self
 
     protected function newAgent(
         Config $config,
-        ContextCollection $sharedContext,
+        ContextCollection $shared_context,
         Connector $connector,
-        EventFactoryInterface $eventFactory,
-        TransactionsStore $transactionsStore): ApmAgent
+        EventFactoryInterface $event_factory,
+        TransactionsStore $transactions_store): ApmAgent
     {
-        if (null === $this->startTime) {
-            $this->startTime = new RequestStartTime(microtime(true));
+        if (null === $this->start_time) {
+            $this->start_time = new RequestStartTime(microtime(true));
         }
 
         if (null === $this->collectors) {
             $this->collectors = new Collection();
         }
 
-        $agent = new Agent($config, $sharedContext, $connector, $eventFactory, $transactionsStore, $this->startTime);
+        $agent = new Agent($config, $shared_context, $connector, $event_factory, $transactions_store, $this->start_time);
```

Comment:
The start time was removed from the `Agent` class so it should no longer be passed in the constructor here.

That also means all of the start time related code in the `AgentBuilder` can be removed.

Reply:
Good catch, I will remove the references 👍

Reply:
You'll need to remove that method call from the `ServiceProvider` as well:

https://github.com/arkaitzgarro/elastic-apm-laravel/blob/fix/async-job-start-time/src/ServiceProvider.php#L99

Reply:
And this is no longer needed in that case:

https://github.com/arkaitzgarro/elastic-apm-laravel/blob/fix/async-job-start-time/src/ServiceProvider.php#L91

Reply:
Remove and commit the changes, because I somehow forgot about it 🤦